PerfNext/app/apis/machines.js

```diff
@@ -15,7 +16,19 @@ module.exports = function(app) {
      * Retrieve all the HW related info about all machines
      */
     app.get('/api/machinesInfo', function(req, res) {
-        res.sendFile(global.appRoot + '/config/master_machine_list.xml');
+    var getOptions = {
+				url: global.APP_DATA.machine_list_url,
+				auth: {
+					'user': global.jenkinsUsername,
+					'pass': global.jenkinsPwd,
+				}
+		};
+		
+		request.get(getOptions, function(error, response, body){
+			console.log('machine.js: Making GET request for getting master_machine_list.xml');
```

Comment:
Should change `master_machine_list.xml` to `machine database file` since this file name could change depending on the `machine_list_url` provided by the PerfNext host.